README.md

```diff
@@ -160,6 +160,16 @@ class I18nEscalatedPrompts < Adhearsion::IVRController
 end
 ```
 
+## Passing URLs for prompts
+
+You might want to pass an URL to an SSML document as a prompt. In that case, wrap the URL in the `#lazy` method and the engine will tag it appropriately.
```

Comment:
Why might I? Mention long prompts and/or caching and potentially serving via Virginia.